target/linux/ath79/dts/ar9344_devolo_magic-2-wifi.dts

```diff
@@ -0,0 +1,181 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9344.dtsi"
+
+/ {
+	model = "Devolo Magic 2 Wifi";
+	compatible = "devolo,magic-2-wifi", "qca,ar9344";
+
+	aliases {
+		led-boot = &plcr;
+		led-failsafe = &plcr;
+		led-running = &plcw;
+		led-upgrade = &plcr;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wlan {
+			label = "devolo:status:wlan";
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy1tpt";
+		};
+
+		plcw: lan_status {
+			label = "devolo:status:dlan";
```

Comment:
So, if I understood it correctly, this would be:
```
		plc_white {
			label = "devolo:white:plc";
```

Reply:
OpenWrt policy is to keep the default LED behavior the same way as vendor firmware does it. This means that if those LEDs are meant for PLC, you cannot use them to indicate boot.

Reply:
Waiting for an answer on the other comments related to this, to decide/close this.